api/proto/teleport/legacy/types/types.proto

```diff
@@ -963,6 +963,26 @@ message CORSPolicy {
   repeated string exposed_headers = 6 [(gogoproto.jsontag) = "exposed_headers,omitempty"];
 }
 
+// IdentityCenterPermissionSet defines a permission set that is available on an
+// IdentityCenter account app
+message IdentityCenterPermissionSet {
+  // ARN is the fully-formed ARN of the Permission Set
+  string ARN = 1 [(gogoproto.jsontag) = "arn,omitempty"];
+
+  // Name is the human-readable name of the permission set
```

Comment:
Nit: it seems that other comments in this file end with a dot, I'd the same here for consistency.